t/03-stream-phases.t

```diff
@@ -0,0 +1,28 @@
+use Test::Nginx::Socket::Lua::Stream;
+
+repeat_each(2);
+plan('no_plan');
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: support preread phase in stream subsystem
+--- stream_config eval
+    "lua_package_path 'src/?.lua;;';"
+--- stream_server_config
+    content_by_lua_block {
```

Comment:
IMHO, this fix is adding support in the preread phase? Why do you test it in the content phase?

Reply:
my mistake, I misunderstood what https://github.com/api7/lua-resty-dns-client/pull/1#discussion_r1040505114 meant.

Reply:
done